examples/Basic_Receive/Basic_Receive.ino

```diff
@@ -1,27 +1,28 @@
 #include <XModem.h>
-XModem xmodem;
+#include <Arduino.h>
+
+bool process_block( byte *data, size_t dataSize) {
```

Comment:
I would prefer this to just be forward declared so that people can start with how the library is setup and run.